.github/workflows/dispatch-npm-engines.yml

```diff
@@ -49,7 +49,7 @@ jobs:
         repositories: ${{ fromJSON(needs.repositories.outputs.matrix) }}
 
     env:
-      NODE_VERSION: "^14.0.0"
+      NODE_VERSION: "^16.0.0"
       NPM_VERSION: "^7.0.0"
```

Comment:
Node 16 uses NPM v8

Reply:
Not all versions.
Node 16 ships npm 8 since 16.11.0. Before it was npm 7.

Reply:
Yes, but all current installation of Node 16 will use NPM v8
And NPM v8 have some important particularities who look relevant.

> npm@8 will refuse to install in node versions it doesn't work with

https://github.com/npm/rfcs/issues/445

This particular behavior is very important to prevent more correlated issues about NPM and Node version in any project.

Reply:
You can still install older node16 versions. I wanted to stay safe. We can indeed raise to npm 8 too :shrug:

Reply:
Also, some distros still ship Node 16 and NPM 7 by default (e.g. Debian). Let's discuss the bump of NPM in another issue/PR.

Reply:
Discussion: https://github.com/nextcloud/standards/discussions/6